sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/AggregationQuerySuite.scala

```diff
@@ -33,101 +33,6 @@ import org.apache.spark.sql.test.SQLTestUtils
 import org.apache.spark.sql.types._
 import org.apache.spark.unsafe.UnsafeAlignedOffset
 
-
-class ScalaAggregateFunction(schema: StructType) extends UserDefinedAggregateFunction {
```

Comment:
@AngersZhuuuu why do we need to move classes? Let's separate refactoring and the fixes when possible.

Reply:
> @AngersZhuuuu why do we need to move classes? Let's separate refactoring and the fixes when possible.

Make a mistake, change back, thanks